src/lib/components/Carousel.svelte

```diff
@@ -1,11 +1,57 @@
 <script>
     import CarouselCard from './CarouselCard.svelte';
+    import { flip } from 'svelte/animate';
+
+    const partners = [
+        { id: 0, name: 'Lorem' },
+        { id: 1, name: 'ipsum' },
+        { id: 2, name: 'dolor' },
+        { id: 3, name: 'sit' },
+        { id: 4, name: 'amet' },
+        { id: 5, name: 'consectetur' },
+        { id: 6, name: 'Lorem' },
+        { id: 7, name: 'ipsum' },
+        { id: 8, name: 'dolor' },
+        { id: 9, name: 'sit' },
+        { id: 10, name: 'amet' },
+        { id: 11, name: 'consectetur' },
+    ];
+
+    let shownPartners = partners.slice(0, 7);
+    let next = 8;
+
+    const mobilePartnersStart = Math.round(partners.length / 3);
+    let shownPartnersMobile = partners.slice(mobilePartnersStart, mobilePartnersStart + 7);
+    let nextMobile = mobilePartnersStart + 8;
+
+    $: setTimeout(() => {
+        shownPartners = [
+            ...shownPartners.slice(1, shownPartners.length),
+            ...partners.slice(next, next + 1),
+        ];
+        next = (next + 1) % partners.length;
+
+        shownPartnersMobile = [
+            ...shownPartnersMobile.slice(1, shownPartnersMobile.length),
+            ...partners.slice(nextMobile, nextMobile + 1),
+        ];
+        nextMobile = (nextMobile + 1) % partners.length;
+    }, 2000);
```

Comment:
This is nice and all, but for the v1, I think we should stick with a simplified [scroll snap](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_scroll_snap) for now.

Reply:
Ay hello! Do I still implement carousel autoplay? Also, do I style the scrollbar?

Reply:
I think we can drop the autoplay for now. And let's use the default scrollbar for this PR in the meantime.